### PR TITLE
TNG-775 unit test for container component

### PR DESCRIPTION
**Pull request by vlapl**

### CODE REVIEW

File: src/components/Container/__tests__/Container.spec.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      props: {
+        fluid: true,
+      },
+    });
+    const container = getByTestId("container");
+    expect(container.classList.contains("w-full")).toBe(true);
+    expect(container.classList.contains("container")).toBe(false);
+  });
+
+  it("should have a specific list of classes when horizontalPadding and verticalPadding are set (default)", () => {
```
Comment (tgolbs) [inline src/components/Container/__tests__/Container.spec.ts:36]: Please remove this test.

File: src/components/Container/__tests__/Container.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      slots: { default: "Some content" },
+    });
+    const containerClasses = getByTestId("container").classList;
+    defaultClasses.forEach(element => {
+      expect(containerClasses.contains(element)).toBe(true);
+    });
+  });
+
+  it("should not have any of the default list of classes when horizontalPadding and verticalPadding are set to false", () => {
+    const { getByTestId } = render(Container, {
```
Comment (tgolbs) [inline src/components/Container/__tests__/Container.spec.ts:47]: Please remove this test.

File: src/components/Container/__tests__/Container.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        verticalPadding: false,
+        horizontalPadding: false,
+      },
+    });
+    const containerClasses = getByTestId("container").classList;
+    defaultClasses.forEach(element => {
+      expect(containerClasses.contains(element)).toBe(false);
+    });
+  });
+});
```
Comment (tgolbs) [inline src/components/Container/__tests__/Container.spec.ts:59]: Please add a test that checks if the passed children are rendered in the default slot.

**Review by tgolbs — CHANGES_REQUESTED** [review]

File: src/components/Container/__tests__/Container.spec.ts (hunk 3 above)
Comment (vlapl) [reply src/components/Container/__tests__/Container.spec.ts:36]: Is this what you meant (are the <span> elements children)?

**Review by tgolbs — APPROVED** [review]

**Pull request merged**